boards/arm/decawave_dwm1001_dev/decawave_dwm1001_dev.dts

```diff
@@ -125,6 +125,16 @@
 	mosi-pin = <20>;
 	miso-pin = <18>;
 	cs-gpios = <&gpio0 17 0>;
+
```

Comment:
@anangl could you take a look at it, changes here leads to a conflict "gen_isr_tables.py: error: multiple registrations at table_index 35 for irq 35 (0x23)". How I see the tree nodes i2c0, spi1, and spi2 use different instances, where is the issue?